kill.c

```diff
@@ -294,6 +295,11 @@ bool is_larger(const poll_loop_args_t* args, const procinfo_t* victim, procinfo_
             debug("pid %d: error reading process name: %s\n", cur->pid, strerror(-res));
             return false;
         }
+        res = get_cmdline(cur->pid, cur->cmdline, sizeof(cur->cmdline));
+        if (res < 0) {
+            debug("pid %d: error reading process cmdline: %s\n", cur->pid, strerror(-res));
+            return false;
+        }
```

Comment:
We don't need cmdline yet, remove here and only do this once, at the end of the function.